.github/workflows/ci.yml

```diff
@@ -47,9 +51,12 @@ jobs:
       with:
         rust-version: stable
     - uses: actions/checkout@v2
+    - name: Install libcap on Ubuntu
+      run: |
+            sudo apt-get install libpcap-dev libpcap0.8
     - name: Install Tarpaulin
       run: cargo install cargo-tarpaulin
     - name: Generate coverage report
       run: cargo tarpaulin --out Xml
     - name: Upload coverage to Codecov
-      uses: codecov/codecov-action@v3
+      uses: codecov/codecov-action@v3
```

Comment:
The last byte of each line should be a newline character. See https://stackoverflow.com/a/729795.

Reply:
Got it! Added a newline after 62

Reply:
Now it has an extra newline character. The original line was

```
"uses: codecov/codecov-action@v3\n"
```

and you first changed it to

```
"uses: codecov/codecov-action@v3"
```

where I pointed out a newline character is missing at the end.

After the most recent change of yours, it became

```
"uses: codecov/codecov-action@v3\n\n"
```

Please remove the superfluous newline character.

Reply:
Done, I think when I saved it in visual studio it automatically adds a new line to the file. So when I added the new line, it added another new line automatically.

Reply:
Should have 63 lines instead of 64 lines now